addons/web_editor/views/snippets.xml

```diff
@@ -439,6 +439,11 @@
                 data-css-property="background-color"
                 data-color-prefix="bg-"/>
 
+        <t t-call="website.snippet_options_border_line_widgets">
```

Comment:
oops, this will break `mass_mailing`. You cannot call `website` stuff in `web_editor`.
If `website` is not installed, you have a traceback :wink: 
![image](https://github.com/user-attachments/assets/bc6ebebd-27f6-4abd-9f70-3883458ac3ed)